packages/v4/src/content/design-guidelines/components/truncate/truncate.md

```diff
@@ -0,0 +1,47 @@
+---
+id: Truncate design guidelines
```

Comment:
This should just say "Truncate"

No need to add "Design guidelines" at the end - it created a new page called "Truncate design guidelines" instead of adding the content to the already existing "Truncate" page haha